app.py

```diff
@@ -33,7 +33,7 @@ def configure_layout(self):
         return APIAccessFrontend(
             apis=[
                 {
-                    "name": "Predict Method",
+                    "name": "generate images",
```

Comment:
Shall it be capital?

Reply:
Let's go with capital for now, and singular

Reply:
@RobertLaurella mind check and approve?